src/components/EventsList/index.tsx

```diff
@@ -1,14 +1,30 @@
 import styled from "@emotion/styled";
+import { useRouter } from "next/router";
+import { useState } from "react";
+
+interface categoryProps {
+  curCategory: number;
+}
 
 export default function EventsList() {
```

Comment:
이 함수의 경우도 return의 타입을 명시해주는 것이 좋은데 명시하지않으면 void인데 저희처럼 컴포넌트일때는 뭐가 있을지 한번 같이 찾아봐야겠네요.
```suggestion
export default function EventsList() {
```

Reply:
이 부분은 더 공부해볼게요!

Reply:
제가 올린 PR 참조해서 하시면 될거 같아요 Emotion.JSX:element로 하면 된다고 나와 있더라구요  !

Reply:
JSX.Element를 쓰는 것과 다른 점이 있을까요?? 👀